lib/Test.rakumod

```diff
@@ -602,6 +602,11 @@ multi sub is-deeply(Mu $got, Mu $expected, $reason = '') is export {
 }
 
 sub throws-like($code, $ex_type, $reason?, *%matcher) is export {
+    for %matcher.kv -> $k, $v {
+        if $v === True|False {
```

Comment:
Considering that `grep` is utilizing multi-dispatch for detecting a `Bool:D` matcher, I would rather suggest using smartmatching for consistency. For example, for readability one defines something like:

```
enum Answer (No => False, Yes => True);
say Yes ~~ Bool:D; # True
say Yes === True;  # False
```

Reply:
I agreed. But I need to check like `if $v ~~ Bool and $v.DEFINITE` because `say Bool ~~ Bool:D; # True`